src/cts/src/TritonCTS.tcl

```diff
@@ -94,19 +94,22 @@ sta::define_cmd_args "clock_tree_synthesis" {[-wire_unit unit]
 
 proc clock_tree_synthesis { args } {
   sta::parse_key_args "clock_tree_synthesis" args \
-    keys {-root_buf -buf_list -wire_unit -clk_nets -sink_clustering_size -num_static_layers\
-          -sink_clustering_buffer -distance_between_buffers -branching_point_buffers_distance -clustering_exponent\
-          -clustering_unbalance_ratio -sink_clustering_max_diameter -sink_clustering_levels -tree_buf\
+    keys {-root_buf -buf_list -wire_unit -clk_nets -sink_clustering_size \
+          -num_static_layers -sink_clustering_buffer \
+          -distance_between_buffers -branching_point_buffers_distance \
+          -clustering_exponent\
+          -clustering_unbalance_ratio -sink_clustering_max_diameter \
+          -sink_clustering_levels -tree_buf\
           -sink_buffer_max_cap_derate -delay_buffer_derate}\
```

Comment:
```suggestion
          -clustering_exponent \
          -clustering_unbalance_ratio -sink_clustering_max_diameter \
          -sink_clustering_levels -tree_buf \
          -sink_buffer_max_cap_derate -delay_buffer_derate} \
```

Reply:
@nmoroze checking for a space before the continuation backslash would be nice

Reply:
though not inside a double-quoted string

Reply:
https://github.com/nmoroze/tclint/issues/17 already opened 🙂